_posts/2016-02-06-office-air-printer.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Copiers and printers puff out particulates
+author: neuroamanda
+---
+
+_Office devices vent some unwelcome chemicals_
+
+Office air can contain many chemical compounds that come from sources like carpet cleaning, furniture, dry cleaned clothes, and office devices.
+These can cause allergic reactions, respiratory illness, headaches, and other symptoms collectively called [sick building syndrome](https://en.wikipedia.org/wiki/Sick_building_syndrome).
+[A recent study](http://dx.doi.org/10.1007/s11356-014-3672-3) specifically looked at the amounts and types of chemicals released by printers and copiers.
+
+When ink and toner are heated during the printing process, trace amounts of their constituent chemicals are released into the surrounding air.
+The amounts depend on the temperature and the frequency of use and operation of the devices.
+In addition, the circuit boards and plastic parts of devices can give off their own emissions.
+Fine aerosol paper particles can also contribute to the office air cocktail of volatile organic compounds (VOCs).
+Volatile refers to the fact that a large number of molecules are released, not that these compounds are inherently dangerous or artificial, though one man-made VOC is probably very familiar: [new car smell](https://en.wikipedia.org/wiki/New_car_smell).
```

Comment:
Not "man-made"! I know you've used "artificial" in the same sentence, but it makes sense to repeat it here.